### PR TITLE
Adding support for several TSQL functions to Spark

### PR DESCRIPTION
**Pull request by GijsVoogd**
Adding support for several TSQL functions to Spark

### CODE REVIEW

File: sqlglot/dialects/hive.py [hunk 1]
```diff
@@ -245,6 +246,7 @@ class Generator(Generator):
             exp.TsOrDsAdd: lambda self, e: f"DATE_ADD({self.sql(e, 'this')}, {self.sql(e, 'expression')})",
             exp.TsOrDsToDate: _to_date_sql,
             exp.TryCast: no_trycast_sql,
+            exp.Repeat: lambda self, e: f"REPEAT({self.format_args(e.this, e.args.get('times'))})",
```
Comment (tobymao) [inline sqlglot/dialects/hive.py:249]: i don't think this is necessary

File: sqlglot/expressions.py [hunk 2]
```diff
@@ -2488,7 +2488,7 @@ class Least(Func):
 
 
 class Length(Func):
-    pass
+    arg_types = {"this": True}
```
Comment (GeorgeSittas) [inline sqlglot/expressions.py:2491]: This is redundant -- (Func < Condition <) Expression already defines arg_types like this.

**Comment by GeorgeSittas** [comment]
@GijsVoogd any plans to finish this?

**Comment by GijsVoogd** [comment]
> @GijsVoogd any plans to finish this?

Yes, will work on it today

File: sqlglot/dialects/hive.py (hunk 1 above)
Comment (GijsVoogd) [reply sqlglot/dialects/hive.py]: removed it, thanks

File: sqlglot/expressions.py (hunk 2 above)
Comment (GijsVoogd) [reply sqlglot/expressions.py]: removed it, thank you

**Comment by GeorgeSittas** [comment]
The reason this test failed is documented [here](https://github.com/tobymao/sqlglot/issues/597). You can change sqlglot.dataframe.sql.functions:809 as follows to fix it:

```python
def repeat(col: ColumnOrName, n: int) -> Column:
    return Column.invoke_expression_over_column(col, glotexp.Repeat, times=lit(n))
```

**Review by GeorgeSittas — APPROVED** [review]
Looks good!

**Comment by GijsVoogd** [comment]
> Looks good!

Thanks for the help George!

**Pull request merged**